forge/routes/api/project.js

```diff
@@ -394,7 +394,14 @@ module.exports = async function (app) {
                     env: request.body.settings.env
                 }
             }
-            const newSettings = app.db.controllers.ProjectTemplate.validateSettings(bodySettings, request.project.ProjectTemplate)
+            let newSettings
+            try {
+                newSettings = app.db.controllers.ProjectTemplate.validateSettings(bodySettings, request.project.ProjectTemplate)
+            } catch (err) {
+                reply.code(400).send({ code: 'env_var_validation', error: `${err.message}` })
+                return
+            }
```

Comment:
Unfortunately that still gives a mismatched error response.

The `validateSettings` itself throws about 9 different errors.  Everyone of these will get the `code` value of `env_var_validation` if we commit this.

Example: bad module specified results in `code` "`env_var_validation`"
![image](https://github.com/flowforge/flowforge/assets/44235289/fb1b56c0-4f69-4e40-8430-c779c94cad25)